src/Integration.UnitTests/LocalServices/RuleSetInspectorTests.cs

```diff
@@ -42,12 +42,15 @@ public void TestInitialize()
             this.serviceProvider.RegisterService(typeof(SVsGeneralOutputWindowPane), this.outputPane);
 
             this.shell = new ConfigurableVsShell();
-            this.shell.RegisterPropertyGetter((int)__VSSPROPID2.VSSPROPID_InstallRootDir, () => this.SonarQubeRuleSetFolder);
+            this.shell.RegisterPropertyGetter((int)__VSSPROPID2.VSSPROPID_InstallRootDir, () => this.VsInstallRoot);
             this.serviceProvider.RegisterService(typeof(SVsShell), this.shell);
 
             this.testSubject = new RuleSetInspector(this.serviceProvider);
 
             Directory.CreateDirectory(this.VsRuleSetsDirectory);
```

Comment:
The fix is relying on the fact that we have a project folder and a share sonar qube folder where we keep all the downloaded rulesets. So I had to apply more constraints on the file paths and ended up creating more helpers and properties (and updated the previous tests for consistency).